zh/backup-to-azblob-using-br.md

```diff
@@ -5,45 +5,70 @@ summary: 介绍如何使用 BR 备份 TiDB 集群数据到 Azure Blob Storage 
 
 # 使用 BR 备份 TiDB 集群数据到 Azure Blob Storage
```

Comment:
preview: https://pingcap-docs-website-scaffold-1hq.pages.dev/zh/tidb-in-kubernetes/dev/backup-to-azblob-using-br